.travis.yml

```diff
@@ -1,10 +1,15 @@
-sudo: required
-dist: trusty
+dist: xenial
```

Comment:
Does this result in us needing newer glibc on some platforms?

Reply:
We used docker to solve this problem.

Reply:
Oh right, so this is the outer bit only, I get it.